Modules/System/SharePoint/src/SharePointEvents.Codeunit.al

```diff
@@ -0,0 +1,12 @@
+codeunit 9112 "SharePoint Events"
+{
+    [IntegrationEvent(false, false)]
+    procedure ProcessSharePointFileMetadata(Metadata: JsonToken; SharePointFile: Record "SharePoint File" temporary)
```

Comment:
This event is missing some documentation.
The event should be an internal procedure.
And the codeunit should get the Access= Public property added.